docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: "3.8"
+services:
+  back-end:
+    build: ./back-end
+    volumes:
+      - ./backend:/visual-programming/back-end
+    ports:
+      - "8000:8000"
+    env_file: ./backend/.env
+    stdin_open: true
+    tty: true
+    command: cd vp && pipenv run python manage.py runserver 0.0.0.0:8000
+  front-end:
+    build: ./front-end
+    volumes:
+      - ./front-end:/visual-programming/front-end
+      # One-way volume to use node_modules from inside image
+      - /visual-programming/front-end/node_modules
+    ports:
+      - "3000:3000"
+    environment:
+      - NODE_ENV=development
+    depends_on:
```

Comment:
The key bit I found here was adding
```
links:
  - back-end
```

This seems to connect the networks between the two Docker containers. The other corresponding change is in the `front-end/package.json` file switching the proxy line to `"proxy": "http://back-end:8000".

Reply:
I missed the last `/` here for the proxy!